presto-main/src/main/java/io/prestosql/sql/planner/LocalDynamicFilter.java

```diff
@@ -138,9 +159,8 @@ private synchronized void addPartition(TupleDomain<String> tupleDomain)
 
         Multimap<String, Symbol> probeSymbols = probeSymbolsBuilder.build();
         PlanNode buildNode = planNode.getRight();
+        // Collect dynamic filters from all build channels
```

Comment:
I think we should collect dynamic filters only for symbols that are listed in `io.prestosql.sql.planner.plan.JoinNode#getDynamicFilters`

Reply:
Update the comment:
```
// Collect dynamic filters for all dynamic filters produced by join
```

Reply:
Updated the comment